python/CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ add_custom_command(
   COMMAND ${PYTHON_EXECUTABLE}
   ARGS "setup.py" "build_ext" "--inplace"
   WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
-  DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/setup.py ${CMAKE_CURRENT_BINARY_DIR}/xcfun_swigPYTHON_wrap.c
+  DEPENDS xcfun-static ${CMAKE_CURRENT_BINARY_DIR}/setup.py ${CMAKE_CURRENT_BINARY_DIR}/xcfun_swigPYTHON_wrap.c
```

Comment:
I tried finding a fix for the shared library, but the way SWIG works (in my limited playing around) again makes it impossible. The `RPATH` on the generated DSO/executable can be fixed by CMake with an easy command, but the problem here is that SWIG takes ownership of this build and CMake know nothing about this target to handle `RPATH`.

Reply:
See my comment below, now CMake is in the driver seat, so possibly the RPATH stuff can be fixed.